user-console/src/main/java/org/pentaho/mantle/client/admin/EmailAdminPanelController.java

```diff
@@ -156,18 +151,41 @@ public void onValueChange( final ValueChangeEvent<Boolean> booleanValueChangeEve
 
     protocolsListBox.addChangeHandler( new ChangeHandler() {
       public void onChange( final ChangeEvent changeEvent ) {
-        emailConfig.setSmtpProtocol( protocolsListBox.getItemText( protocolsListBox.getSelectedIndex() ) );
+        String protocol = protocolsListBox.getValue( protocolsListBox.getSelectedIndex() );
+        smtpPanel.setVisible( protocol.contains( "SMTP" ) );
+        useSSLCheckBox.setVisible( protocol.contains( "SMTP" ) );
+        useStartTLSCheckBox.setVisible( protocol.contains( "SMTP" ) );
+        if ( protocol.equalsIgnoreCase( "graph_api" ) ) {
+          authenticationTypeLB.setSelectedIndex( 0 );
+          emailConfig.setAuthMechanism( authenticationTypeLB.getValue( 0 ) );
+          emailConfig.setAuthenticate( true );
+          authenticationTypeLB.setEnabled( false );
+          oauthPanel.setVisible( true );
+          passwordPanel.setVisible( false );
+          authenticationPanel.setVisible( true );
```

Comment:
Don't these panels need to be hidden when `! protocol.equalsIgnoreCase( "graph_api" )`?